### PR TITLE
feat(siren): add create alert with namespace api

### PR DESCRIPTION
**Pull request by mabdh**
Related with https://github.com/odpf/siren/issues/163

### CODE REVIEW

File: odpf/siren/v1beta1/siren.proto [hunk 1]
```diff
@@ -287,6 +287,19 @@ service SirenService {
...
   }
 
+  rpc CreateAlertsWithNamespace(CreateAlertsWithNamespaceRequest) returns (CreateAlertsWithNamespaceResponse) {
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "create alerts with namespace";
+      tags: "Alert";
+    };
+
+    option (google.api.http) = {
+      post: "/v1beta1/alerts/{provider_type}/{provider_id}/{namespace_id}"
```
Comment (ravisuhag) [inline odpf/siren/v1beta1/siren.proto:297]: @mabdh what blocks us from having `POST` on alerts with path `alerts/` and the rest of the parameters can be accepted with the body. Do we have a completely different structure of payload for each provider?

Comment (mabdh) [reply odpf/siren/v1beta1/siren.proto:297]: @ravisuhag problem is we need to add this api in Cortex `webhook_config` url to send alerts and the body can't be configured and does not have namespace information, what we can do is just assign different url for different tenant.

Comment (mabdh) [reply odpf/siren/v1beta1/siren.proto:297]: as an alternative, maybe we could just use `/v1beta1/alerts/{namespace_id}` ? considering provider type and provider id would be unique to each namespace?

Comment (mabdh) [reply odpf/siren/v1beta1/siren.proto:297]: > Do we have a completely different structure of payload for each provider?
I think we need to expect the payload to be different for each provider and design the API to be extendable. We did that so each provider has a function to parse the consumed request body.

Comment (ravisuhag) [reply odpf/siren/v1beta1/siren.proto:297]: Make sense. I think lets go ahead with this for now. And we can revisit this later.

**Review by StewartJingga — APPROVED** [review]

**Pull request merged**